lib/fabrik/DirectorManager.js

```diff
@@ -315,8 +314,8 @@ class DirectorManager extends BaseManager {
       name: deploymentName,
       director_uuid: this.director.uuid,
       releases: this.releases,
-      compilation: infrastructure.compilation,
-      disk_pools: infrastructure.disk_types,
+      compilation: this.director.config.infrastructure.compilation,
+      disk_pools: this.director.config.infrastructure.disk_types,
```

Comment:
comments similar to [above](https://github.com/SAP/service-fabrik-broker/pull/7/files#r133368904)